R/deployApp.R

```diff
@@ -237,7 +237,7 @@ deployApp <- function(appDir = getwd(),
   }
 
   # invoke pre-deploy hook if we have one
-  runDeploymentHook(appPath, "rsconnect.pre.deploy", verbose = verbose)
+  runDeploymentHook(appDir, "rsconnect.pre.deploy", verbose = verbose)
```

Comment:
Yeah, this is a change, and the documentation for the hooks is murky: 

>   \item{\code{rsconnect.pre.deploy}}{A function to run prior to deploying content; it receives as an argument the path to the content that's about to be deployed.}

What does "content about to be deployed" mean? (especially if you've used `appFiles` or friends). 

My guess is that this won't affect any code in the wild, but I'll add a news bullet.

FWIW I think the thematic usage would break when `appPath` is not a directory.